docs/src/examples.md

```diff
@@ -1,59 +1,67 @@
 # Entropies.jl Examples
 
-## Nearest neighbor direct entropy example
+## Direct entropy (nearest neighbors)
```

Comment:
You mean Indirect entropy here, right...? The only nearest neighbor entropy that is direct that we could have is one from `NaiveKernel`.

Reply:
(go ahead and merge once this is clariffied)

Reply:
> You mean Indirect entropy here, right...? The only nearest neighbor entropy that is direct that we could have is one from NaiveKernel.

Yes, good catch.